### PR TITLE
Copywriting improvement when probe does not exist

### LINKED ISSUE #242: Show info message when there are no probes
Empty Probes Page

### PR DESCRIPTION
**Pull request by samsunoor**
# Symon Pull Request (PR)

### What issue does this pr address
1. This PR resolves #242 which improves copywriting when current user does not have any probes configured.
2. Telling user does not have probes and add link to [monika quick start guide](https://monika.hyperjump.tech/quick-start)

### How did you implement
1. improve default home page on `src/client/pages/Home/Home.tsx`

### How to test
1. Clear development data and run project
```
# remove development db
$ rm src/server/prisma/dev.db

# run db migration
$ npx prisma migrate dev --preview-feature
$ npx prisma generate
$ npm run start
```
2. Check `localhost:4000`
3. Login default user
4. Should see the screen as below
image

### CODE REVIEW

**Review by raosan — APPROVED** [review]

File: src/client/pages/Home/Home.tsx [hunk 1]
```diff
@@ -76,7 +75,15 @@ export const Home: FC = () => {
 
   return (
     <Layout>
-      <div>Home</div>
+      <strong>You don&apos;t have any probes yet.</strong>
+      <p style={{ marginTop: "1rem" }}>
```
Comment (nicnocquee) [inline src/client/pages/Home/Home.tsx:79]: use `className="mt-8"` since we're using tailwind.

File: src/client/pages/Home/Home.tsx [hunk 2]
```diff
@@ -76,7 +75,15 @@ export const Home: FC = () => {
...
+        Run Monika with Symon configuration to see the probes here. Please check
+        out{" "}
+        <a href="https://monika.hyperjump.tech/quick-start">
```
Comment (nicnocquee) [inline src/client/pages/Home/Home.tsx:82]: use `className="underline"` instead of `<u>`

**Review by nicnocquee — CHANGES_REQUESTED** [review]

File: src/client/pages/Home/Home.tsx [hunk 3]
```diff
@@ -76,7 +75,18 @@ export const Home: FC = () => {
 
   return (
     <Layout>
-      <div>Home</div>
+      <strong>You don&apos;t have any probes yet.</strong>
```
Comment (kevinhermawan) [inline src/client/pages/Home/Home.tsx:78]: Why use `&apos;` instead of `'` symbol?

Comment (nicnocquee) [reply src/client/pages/Home/Home.tsx:78]: i think eslint or something warns to escape some symbols.

Comment (samsunoor) [reply src/client/pages/Home/Home.tsx:78]: true, I use it since the eslint would return error about not escaped symbol, i.e., `'`

**Review by nicnocquee — APPROVED** [review]

**Pull request merged**